off_chain_data/application-go/go.mod

```diff
@@ -0,0 +1,22 @@
+module offChainData
+
+go 1.22.7
+
+toolchain go1.23.0
```

Comment:
I think I would recommend setting the minimum required Go version to 1.22.0 and removing the toolchain so it builds with the locally installed version of Go, provided it is at least the minimum required version. You can do that by running:

```shell
go get go@1.22.0 toolchain@none
```

Reply:
Thx. Will do.